nav2_dummy_controller/src/dummy_controller.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2018 Intel Corporation
+//
```

Comment:
What is the point of dymmy controller?  Do we need it in master?

Reply:
I second this

Reply:
@mhpanah @SteveMacenski I've found it really useful having dummy modules for development (FakeLocalization, Dummy controller & planner). What problem do you see with having these?

Reply:
It might be useful, but its not a production tool. If you wanted to include a dummy controller for CI or development, then it should go into the system tests package and not outside as an existing package. Fake localization is a needed tool to do full simulations in gazebo of an actual robotics system, where this is a placeholder to make testing and evaluation easier but not a production-level test or simulation.

Reply:
I’m not convinced that putting this dummy controller and planner on the master adds any value.   If this code segment is meant to be used as an example, then it could perhaps be explained in readme files or if it is for development then as @SteveMacenski said it could either go into the system tests package or on a different branch

Reply:
Agree on moving to System Tests.